spec/unit/platform/query_helpers_spec.rb

```diff
@@ -114,6 +114,26 @@
   end
 end
 
+describe "Chef::Platform#supports_msi?" do
+  let(:key) { "System\\CurrentControlSet\\Services\\msiserver" }
+  let(:key_query_value) { 0x0001 }
+  let(:access) { key_query_value }
+  let(:error_code) { 0 }
+
+  it "returns false early when not on windows" do
+    allow(ChefConfig).to receive(:windows?).and_return(false)
+    expect(Chef::Platform.supports_msi?).to be false
+  end
+
+  it "returns true when the registry key exists" do
+    allow(ChefConfig).to receive(:windows?).and_return(true)
+    allow(Win32::Registry::HKEY_LOCAL_MACHINE).to receive(:open).
```

Comment:
Hmm, the test seems to succeed on Windows though?

Travis failure shows Win32 undefined - is that a separate issue?
  1) Chef::Platform#supports_msi? returns true when the registry key exists
     Failure/Error: allow(Win32::Registry::HKEY_LOCAL_MACHINE).to receive(:open).
     NameError:
       uninitialized constant Win32